docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -108,6 +108,20 @@ permission to restart the Nginx service as root. In Teleport 16.4.0 and later,
 the default shell for a created user can be configured with `create_host_user_default_shell`. 
 Otherwise the host's default shell will be used.
 
+<Admonition type="note">
+
+When using the terraform provider, the values for `create_host_user_mode` are represented numerically
+as reflected in the
+[role reference](/reference/terraform-provider/resources/role/#nested_schema-for-specoptions).
```

Comment:
Just realized I didn't double check that this is a working link, so I'll do that before this lands

Reply:
The underscore in the jump link needed to be a dash, but this works as expected :+1: